### PR TITLE
Add `levelView.drawHintPath` API for more specific in-game hints

### PR DESCRIPTION
**Pull request by joshlory**
Resolves https://github.com/code-dot-org/craft/issues/321.  Sample usage:

```js
gameController.levelView.drawHintPath([[5, 9], [5, 7], [6, 7], [6, 5]]);
```

screen shot 2017-10-17 at 3 09 36 pm

### CODE REVIEW

File: src/js/game/LevelMVC/LevelView.js [hunk 1]
```diff
@@ -1392,11 +1392,37 @@ module.exports = class LevelView {
     this.selectionIndicator.y = -55 + 43 + 40 * y;
   }
 
+  /**
+   * @param {Array<Array<int>>} gridSpaces An array of x and y grid coordinates.
+   */
+  drawHintPath(gridSpaces) {
+    const hintPath = this.game.add.bitmapData(400, 400);
```
Comment (Hamms) [inline src/js/game/LevelMVC/LevelView.js]: can we infer these values?

**Comment by joshlory** [comment]
Yes!  If I wanted to call code when a hint was displayed, where would I start?

**Comment by Hamms** [comment]
A hint as in an authored hint? Oh gosh, probably the [authored hint controller object](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/authoredHints.js), or possibly [the redux store it manages](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/redux/authoredHints.js)

**Comment by kiki-lee** [comment]
Woah, this is awesome!

**Pull request merged**